src/runtime/Object.h

```diff
@@ -643,17 +659,35 @@ class ObjectGetResult {
         return m_isDataProperty;
     }
 
+    bool isDataAccessorProperty() const
+    {
+        ASSERT(hasValue());
+        return !m_isPlainDataProperty;
+    }
+
     // http://www.ecma-international.org/ecma-262/5.1/#sec-8.10.4
     Value toPropertyDescriptor(ExecutionState& state, const Value& receiver);
 
+    ObjectPropertyNativeGetterSetterData* nativeGetterSetterData()
+    {
+        ASSERT(isDataAccessorProperty() && isDataProperty());
+        return m_nativeGetterSetterData.m_nativeGetterSetterData;
+    }
+
 private:
     bool m_hasValue : 1;
     bool m_isWritable : 1;
     bool m_isEnumerable : 1;
     bool m_isConfigurable : 1;
     bool m_isDataProperty : 1;
+    bool m_isPlainDataProperty : 1;
```

Comment:
Woud you please add some comments about `m_isDataProperty` and `m_isPlainDataProperty` members?
I'm confused because these members looks very similar.

Reply:
You are right!. I added comment & rename the variable